addons/cba_settings/cba_settings.sqf

```diff
@@ -64,6 +64,7 @@ acex_volume_lowerInVehicles = true;
 
 // ZEN
 zen_editor_disableLiveSearch = true;
+zen_common_disableGearAnim = true;
```

Comment:
Above the editor setting, should be alphabetical if possible. Also separate with empty line because it's from a different component.

Reply:
Made the requested changes.